content/en/docs/installation/upgrading/remove-deprecated-apis.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Removing Deprecated API Resources"
+linkTitle: "Removing Deprecated API Resources"
+weight: 20
+type: "docs"
+---
+
+We have deprecated the following cert-manager APIs: 
+- `cert-manager.io/v1alpha2`
+- `cert-manager.io/v1alpha3`
+- `cert-manager.io/v1beta1`
+- `acme.cert-manager.io/v1alpha2`
+- `acme.cert-manager.io/v1alpha3`
+- `acme.cert-manager.io/v1beta1`
+These APIs will be removed in cert-manager `v1.6.0`.
```

Comment:
suggestion: tested locally: this doesn't render correctly as a list without spaces:

```suggestion
We have deprecated the following cert-manager APIs: 

- `cert-manager.io/v1alpha2`
- `cert-manager.io/v1alpha3`
- `cert-manager.io/v1beta1`
- `acme.cert-manager.io/v1alpha2`
- `acme.cert-manager.io/v1alpha3`
- `acme.cert-manager.io/v1beta1`

These APIs will be removed in cert-manager `v1.6.0`.
```